pkg/project/admission/requestlimit/admission.go

```diff
@@ -61,6 +61,9 @@ var _ = oadmission.Validator(&projectRequestLimit{})
 
 // Admit ensures that only a configured number of projects can be requested by a particular user.
 func (o *projectRequestLimit) Admit(a admission.Attributes) (err error) {
+	if o.config == nil {
```

Comment:
I really wish these could get out of the chain if they had a nil config (if the func passed to admission.RegisterPlugin could return something that said "actually, I don't want to be in the chain).